pkg/server/nodemanager/node_scaler.go

```diff
@@ -102,8 +102,9 @@ func (s *BindingNodeScaler) createNodeForPod(pod *api.Pod) *api.Node {
 	// until dispatch (just launch with the default size), otherwise,
 	// if we have a fixed size disk of non-default size, size the disk
 	// here and also take care of matching disk sizes
+
 	node.Spec.Resources.VolumeSize = s.defaultVolumeSize
-	if s.fixedSizeVolume && pod.Spec.Resources.VolumeSize != "" {
+	if pod.Spec.Resources.VolumeSize != "" { // TODO: consider if && s.fixedSizeVolume is needed here
```

Comment:
@ldx @henryprecheur for AWS `s.fixedSizeVolume` is always set to false. Which basically means that pod annotation `pod.elotl.co/volume-size: "X Gi"` has no effect, which may be confusing for user (it isn't also specified nowhere in docs for which cloud providers this annotation works). I needed to manually set volume size to test if the size is an issue and noticed that. Should we address it in a separate issue or is this desired behavior?

Reply:
This annotation does work on AWS. However, resizing happens in the pod controller, not in the node controller when `fixedSizeVolume` is false (i.e. on AWS).